Automattic/wp-calypso
# Add Basic Jetpack Cloud Activity Log

#### Changes proposed in this Pull Request

* Use the ActivityCardList component in the ActivityLogPage

#### Testing instructions

1. Navigate to `/backups/activity`
2. Use the activity log. Verify that behavior of filtering roughly matches `https://wordpress.com/activity-log/:site`
